packages/ra-input-rich-text/src/RichTextInput.spec.tsx

```diff
@@ -23,4 +23,22 @@ describe('<RichTextInput />', () => {
             );
         });
     });
+    it("should update its content with the ref editor's commands", async () => {
```

Comment:
Wrap a `describe('ref')` around the tests for that prop.

Reply:
done